server/commands/project_runner.go

```diff
@@ -1,17 +1,4 @@
-// Copyright 2017 HootSuite Media Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the License);
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//    http://www.apache.org/licenses/LICENSE-2.0
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an AS IS BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-// Modified hereafter by contributors to runatlantis/atlantis.
-
-package events
+package commands
```

Comment:
personally not a fan of this name because it gets overloaded with like cli commands. Would love to change this concept in atlantis going forward.

What if we use workflow instead of command?

```
server
└── workflow
    └── project
        ├── locker.go
        └── runner.go

2 directories, 2 files                                        
```